com.her01n.BatteryInfo.json

```diff
@@ -0,0 +1,85 @@
+{
+  "app-id": "com.her01n.BatteryInfo",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "battery-info",
+  "modules": [
+    {
+      "name": "gc",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/ivmai/bdwgc/releases/download/v8.2.4/gc-8.2.4.tar.gz",
+          "sha256": "3d0d3cdbe077403d3106bb40f0cbb563413d6efdbb2a7e1cd6886595dec48fc2"
+        }
+      ]
+    },
+    {
+      "name": "guile",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://ftp.gnu.org/gnu/guile/guile-3.0.9.tar.gz",
+          "sha256": "18525079ad29a0d46d15c76581b5d91c8702301bfd821666d2e1d13726162811"
+        }
+      ]
+    },
+    {
+      "name": "guile-lib",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--with-guile-site"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://download.savannah.nongnu.org/releases/guile-lib/guile-lib-0.2.7.tar.gz",
+          "sha256": "e4ef3b845f121882c7c0cf04f81a1cb8fd360c6f64b56b868de5546214f904de"
+        }
+      ]
+    },
+    {
+      "name": "g-golf",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--with-guile-site"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "http://ftp.gnu.org/gnu/g-golf/g-golf-0.8.0-rc-2.tar.gz",
+          "sha256": "0b21c82c4c09e3f8736994f655bb72ee41210ad927701737b8d89e6f7f2b54e1"
+        }
+      ]
+    },
+    {
+      "name": "battery-info",
+      "buildsystem": "simple",
+      "build-commands": [
+        "make PREFIX=/app mos",
+        "make PREFIX=/app install",
+        "mkdir -p /app/lib",
+        "ln -s /usr/lib/$HOSTTYPE-$OSTYPE/libgio-2.0.so.0 /app/lib/libgio-2.0.so",
+        "ln -s /usr/lib/$HOSTTYPE-$OSTYPE/libgirepository-1.0.so.1 /app/lib/libgirepository-1.0.so",
+        "ln -s /usr/lib/$HOSTTYPE-$OSTYPE/libglib-2.0.so.0 /app/lib/libglib-2.0.so",
+        "ln -s /usr/lib/$HOSTTYPE-$OSTYPE/libgobject-2.0.so.0 /app/lib/libgobject-2.0.so"
```

Comment:
Ah the main binary runs guile -L and -c and it can't find libraries via ld.so

Reply:
> his is strange because the symlink is present when one enters the sdk with

plain sonames are stripped from org.gnome.Platform for reasons mentioned above. (only used at build time)

Reply:
on an RPM distro, the .so are in the -devel package, and one a DEB distro, they are in the -dev package. Same thing.

Reply:
Seems to me like a bug in guile. Dynamic library load fails in case .so files are not installed.
https://pastebin.com/bby894QE

Reply:
I have found it open for two years. https://debbugs.gnu.org/cgi/bugreport.cgi?bug=49232